### ISSUE TITLE
Make a standardised yaml file

### ISSUE THREAD

**Issue by alinaselega** [Neuroglycerin/neukrill-net-work#51]
Want to have two yaml file templates for the current best model. One template should be for training from scratch (and so should specify the model's architecture) and one - for continuing to train from a loaded .pkl file. In both templates the following should be configurable through the .json file: batch size, learning rate schedule, momentum, dropouts.
